ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -447,29 +431,27 @@ export default React.memo<Props>(function VideoJs(props: Props) {
         type: type,
       });
     });
-  }, [source]);
+  }, [source, reload]);
 
   return (
-    reload && (
```

Comment:
I never understood why this was here... the value of reload is initialized to a string (`initial`) and is updated to a `number`, I don't believe it would ever return falsy.

It's also a `useState` so modifying it's value triggers a re-render as anyways as well. Maybe if it's value was initialized to a falsy value I would understand it (i.e. it saves a few dom elements from being rendered), but in it's current state, I don't get it.